x-pack/platform/plugins/shared/stack_connectors/public/connector_types/webhook/webhook.tsx

```diff
@@ -29,17 +30,27 @@ export function getConnectorType(): ConnectorTypeModel<
       defaultMessage: 'Webhook data',
     }),
     validateParams: async (
-      actionParams: WebhookActionParams
+      actionParams: WebhookActionParams,
+      connectorConfig?: Record<string, unknown>
     ): Promise<GenericValidationResult<WebhookActionParams>> => {
+      const webhookMethod: WebhookMethods = connectorConfig?.method
+        ? (connectorConfig.method as unknown as WebhookMethods)
```

Comment:
Can we avoid this casting? Isn't `connectorConfig` is of a `WebhookConfig` type?